src/JPEGView/HelpersGUI.cpp

```diff
@@ -425,6 +425,15 @@ namespace HelpersGUI {
 		if (bOutOfMemory) {
 			_tcscat_s(buff, BUF_LEN, _T("\n"));
 			_tcscat_s(buff, BUF_LEN, CNLS::GetString(_T("Reason: Not enough memory available")));
+		} else if (nFileLoadError == FileLoad_LoadError) {
+			LPCTSTR sEnding = _tcsrchr(sFailedFileName, _T('.'));
+			if (sEnding != NULL) {
+				sEnding += 1;
+				if (_tcsicmp(sEnding, _T("JXL")) == 0) {
+					_tcscat_s(buff, BUF_LEN, _T("\n"));
+					_tcscat_s(buff, BUF_LEN, CNLS::GetString(_T("JXL decoding requires the Microsoft Visual C++ Redistributable")));

```

Comment:
I like the elegant solution of displaying the error message directly on JPEGView window.  Very cool way of implementing it.

Took me awhile to break a Windows system enough to not have the vcruntime140.dll, so I could test this use case